google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPool.java

```diff
@@ -1794,7 +1795,7 @@ private boolean canCreateSession() {
     }
   }
 
-  private void createSessions(final int sessionCount) {
+  private void createSessions(final int sessionCount, boolean initialization) {
```

Comment:
Why do we name this as from `distributeOverChannels` to `initialization`? `initialization` seems not to have a clear meaning.

Reply:
Good point, changed. I had named it `initialization` because the session pool only calls this with `true` during initialization, but naming it by what it actually does instead of when it's called, is a lot more helpful.